internal/routes/server.go

```diff
@@ -169,13 +169,6 @@ func Server(ctx context.Context, cfg *config.ServerConfig, db *database.Database
 			sub.Handle("/login/change-password", loginController.HandleShowChangePassword()).Methods("GET")
 			sub.Handle("/login/change-password", loginController.HandleSubmitChangePassword()).Methods("POST")
 			sub.Handle("/account", loginController.HandleAccountSettings()).Methods("GET")
-
-			// Verifying email requires the user is logged in
-			sub = r.PathPrefix("").Subrouter()
-			sub.Use(requireAuth)
-			sub.Use(rateLimit)
```

Comment:
still feels like we should keep the rate limiter?

Reply:
see line 164. This merges it with the above subrouter which still has rate limiting